package.json

```diff
@@ -20,6 +20,9 @@
   "bugs": {
     "url": "https://github.com/mozilla/shield-studies-addon-template/issues"
   },
+  "engines": {
+    "node": ">=8.9.0"
```

Comment:
I know we [require](https://github.com/mozilla/shield-studies-addon-template#testing) Node 7.6+ for async/await, but is there a specific reason why we require >= 8.9.0? I read that it's the LTS for Node v8. Just curious mostly.